db/src/db.rs

```diff
@@ -1108,9 +1103,7 @@ impl PartitionMap {
 mod tests {
     extern crate env_logger;
 
-    use std::borrow::{
-        Borrow,
-    };
+    use std::borrow::Borrow;
```

Comment:
Eh?  Generally I've been trying to migrate to the `{\n}` style, since it's consistent.

Reply:
Sure!